### ISSUE TITLE
Sort Y axis

### ISSUE THREAD

**Issue by ruthpozuelo** [microsoft/charticulator#712]
**Is your feature request related to a problem? Please describe.**
I am trying to sort the Y-axis on a rank chart, and the only way I have been able to do it is by changing the order of the Range values.
If I do that, to make it stick, I have to also untick Auto min and max and suddenly the values get hardcoded.

I need to be able to sort without hardcoding (ok to set min and max values if it is not possible to do a normal sort).
![image](https://user-images.githubusercontent.com/19476550/131362237-d2260d1b-6112-4af1-995b-b673177a8ef0.png)

**Comment by zBritva** [comment]
It seems you want to reverse numerical axis in plot segemnt.

**Comment by ruthpozuelo** [comment]
Yes, is that possible? Also I am trying to figure out why the links wont follow the years...😩😩

**Comment by zBritva** [comment]
1. No. Only option is flipping plotsegment but gives wrong rendering for X axis.
2. Check the sorting on the visual:

I suspect it uses "Rank" for sorting:

![image](https://user-images.githubusercontent.com/10897951/131370824-5d5b4eb3-ad61-43d1-ae51-fc034c85e009.png)

**Comment by ruthpozuelo** [comment]
OmG thanks @zBritva , I didnt realized that sorting in power bi affected the sort in charticulator (although it makes sense)!

For the issue, in case it helps someone until fixed, if you convert the rank to category instead of number, it can be sorted properly and with the right precision in the axis:
![image](https://user-images.githubusercontent.com/19476550/131378153-5f3f56e9-2f1e-4773-b170-f06a8c048eaf.png)
